### PR TITLE
Page/active-favorites

### PR DESCRIPTION
**Pull request by ssmartha**
See commits 
Added saved properties & my properties page with handlers & logic!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+import { useAuth } from "./context/auth-context";
 
 function App() {
+  const { user, userType} = useAuth();
 
   return (
     <div>
-      <Navbar />
+      {user && userType === "home-seeker" ? <HomeSeekerNavbar /> :
+       user && userType === "landlord" ? <LandlordNavbar /> :<UnauthenticatedNavbar />}
```
Comment (paulotijero) [inline src/App.js:18]: I don't understand why it has 2 user states if it has the same value.

Reviewing the [auth-context (R8-R9)](https://github.com/andresnb/react-get-that-home/pull/9/files#diff-50d25f7adf348217c540867a8937453cd2a126a2abdf2681a9e4a838a51f9714R8-R9) makes me understand that the user is for testing purposes only, I would recommend simplifying this and using a single variable.

**Review by paulotijero — CHANGES_REQUESTED** [review]
This PR contains changes from previous PR #8, so I'll block this one until the changes from PR #8  are first pushed to the main branch and this PR is resolved with a `rebase/merge` to provide clearer code.

File: src/App.js (hunk 1 above)
Comment (andresnb) [reply src/App.js:18]: This should be better as it only checks for user once and then looks for user type if it exists

 `{user ? userType === "home-seeker" ? <HomeSeekerNavbar /> : <LandlordNavbar /> : <UnauthenticatedNavbar />}`

**Review by andresnb — COMMENTED** [review]
See comments for line change in App.js

**Pull request merged**